app/src/main/kotlin/maru/app/MaruApp.kt

```diff
@@ -168,10 +169,17 @@ class MaruApp(
       NewBlockHandlerMultiplexer(followerBlockHandlers)
     val adaptedBeaconBlockImporter = SealedBeaconBlockHandlerAdapter(blockImportHandlers)
 
+    val qbftForkTimestamp =
```

Comment:
Maybe we should introduce this as a dedicated method rather than exposing `getAllForks`?

Reply:
Yeah good idea, it did feel a bit wrong exposing that method. I'll add that